businessRules/users.js

```diff
@@ -102,40 +106,24 @@ function updateUser(userId, key, value) {
 }
 
 function deleteUser(userId) {
-    let users = getUsers();
+    const users = getUsers();
 
-    const user = users.find((user) => userId === user.id);
+    const updatedUsers = users.filter((user) => userId !== user.id);
 
-    if (!user) {
-        throw new Error('Error. No user with such ID found.');
-    }
-
-    let updatedUsers = users.filter((user) => userId !== user.id);
-
-    setIdeas(updatedUsers);
+    setUsers(updatedUsers);
 }
 
 function checkIfLoggedIn(userLoginSession) {
-    if (!userLoginSession) {
-        throw new Error('Error. There was no users session data provided.');
-    }
-
-    let users = getUsers();
-
-    const user = userLoginSession
-        ? users.find((user) => user.session === userLoginSession)
-        : null;
-
-    if (!user) {
-        throw new Error('You are not logged in.');
-    }
+    const user = findLoggedInUser(userLoginSession);
     return user;
 }
 
 module.exports = {
     registerUser,
+    findRegisteredUser,
+    findLoggedInUser,
     login,
-    usersList,
+    usersList: getUsers,
```

Comment:
Is `usersList` a setter or a getter? Can you improve its name please

Reply:
Fixed.